cax/cax_transfer.json

```diff
@@ -0,0 +1,27 @@
+[
```

Comment:
Can't. Need separate config for a separate transfer-only process that happens in post-script of DAG. 

(cax_stash replication is also running a separate .json, but just not under version control.)